pages/osx/softwareupdate.md

```diff
@@ -0,0 +1,19 @@
+# softwareupdate
+
+> A tool for updating MacOS App Store apps via the command line.
+
+- List all available updates:
+
+`softwareupdate -l`
+
+- Download and install all updates:
+
+`softwareupdate -ia`
+
+- Download and install all recommended updates:
+
+`softwareupdate -ir`
+
+- Download and install a specific app:
+
+`softwareupdate -i {{update_name}}`
```

Comment:
Should it be {{app_name}} instead of {{update_name}} ?

Reply:
I thought it would be better to use update_name over app_name because the OS can also be updated through this method.

Reply:
I see. Alright.